client/src/controllers/TabsController.test.js

```diff
@@ -0,0 +1,82 @@
+import { Application } from '@hotwired/stimulus';
```

Comment:
This file needs some formatting, best to run `npm run format` before committing or enable the pre-commit hooks.

https://docs.wagtail.org/en/stable/contributing/developing.html#automatically-lint-and-code-format-on-commits